static/js/components/Toast.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+export default class Toast extends React.Component {
```

Comment:
Toast is a standard name for a popup notification to a user: https://material.google.com/components/snackbars-toasts.html